Shared/Samples/Show grid/ShowGridView.swift

```diff
@@ -251,15 +251,17 @@ private extension ShowGridView {
 }
 
 private extension ArcGIS.Grid.LabelPosition {
-    static var allCases: [Self] = [
-        .allSides,
-        .center,
-        .topLeft,
-        .topRight,
-        .bottomLeft,
-        .bottomRight,
-        .geographic
-    ]
+    static var allCases: [Self] {
```

Comment:
Why not use `let` like other cases?

Reply:
We generally favor using computed properties, see Phil's comment [here](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/24#discussion_r921690047). 

When it comes to [Plain-Old-Data](https://en.wikipedia.org/wiki/Passive_data_structure)-like types, such as `Int` or `Double` or `Char`, because the compiler optimization [results in the same assembly code](https://forums.swift.org/t/static-computed-property-vs-static-let-constant/20968/3) for `static let` vs `static var` computed, it doesn't matter too much which to use. For more complex data types such as collections and strings, we prefer computed var.

I can change them all to `static var`